doc/rados/operations/monitoring-osd-pg.rst

```diff
@@ -187,99 +186,100 @@ OSDs to establish agreement on the current state of the placement group
-``active``, and preferably in the ``clean`` state. To see the status of all
-placement groups, execute:
+If one of these circumstances causes Ceph to echo ``HEALTH WARN``, don't
+panic. In many cases, the cluster will recover on its own. In some cases, however, you
+might need to take action. An important aspect of monitoring PGs is to check their
+status as ``active`` and ``clean``: that is, it is important to ensure that, when the
+cluster is up and running, all PGs are ``active`` and (preferably) ``clean``.
+To see the status of every PG, run the following command:
 
 .. prompt:: bash $
 
-	ceph pg stat
+    ceph pg stat
 
-The result should tell you the total number of placement groups (x), how many
-placement groups are in a particular state such as ``active+clean`` (y) and the
+The output provides the following information: the total number of PGs (x), how many
+PGs are in a particular state such as ``active+clean`` (y), and the
 amount of data stored (z). ::
 
-	x pgs: y active+clean; z bytes data, aa MB used, bb GB / cc GB avail
+    x pgs: y active+clean; z bytes data, aa MB used, bb GB / cc GB avail
 
-.. note:: It is common for Ceph to report multiple states for placement groups.
+.. note:: It is common for Ceph to report multiple states for PGs.
 
-In addition to the placement group states, Ceph will also echo back the amount of
+Here Ceph echoes back not only the PG states, but also 
 storage capacity used (aa), the amount of storage capacity remaining (bb), and the total
-storage capacity for the placement group. These numbers can be important in a
-few cases: 
+storage capacity of the PG. These values can be important in a
+few cases:
 
-- You are reaching your ``near full ratio`` or ``full ratio``. 
-- Your data is not getting distributed across the cluster due to an 
+- You are reaching your ``near full ratio`` or ``full ratio``.
+- Your data is not being distributed across the cluster due to an 
   error in your CRUSH configuration.
 
 
 .. topic:: Placement Group IDs
 
-   Placement group IDs consist of the pool number (not pool name) followed 
-   by a period (.) and the placement group ID--a hexadecimal number. You
-   can view pool numbers and their names from the output of ``ceph osd 
-   lspools``. For example, the first pool created corresponds to
-   pool number ``1``. A fully qualified placement group ID has the
+   PG IDs consist of the pool number (not the pool name) followed 
+   by a period (.) and the PG ID-- (a hexadecimal number). You
+   can view pool numbers and their names from in the output of ``ceph osd 
+   lspools``. For example, the first pool that was created corresponds to
+   pool number ``1``. A fully qualified PG ID has the
    following form::
-   
-   	{pool-num}.{pg-id}
-   
-   And it typically looks like this:: 
-   
-	1.1f
-   
 
-To retrieve a list of placement groups, execute the following:
+       {pool-num}.{pg-id}
+
+   It typically resembles the following:: 
+
+    1.1f
```

Comment:
This is nitpicky even for me, but I suggest a longer number than `1f`, say `1.1701b`